.github/workflows/snyk-push.yml

```diff
@@ -0,0 +1,25 @@
+name: Run Snyk monitor on push
+
+on:
+  push:
+    branches:
+      - master**
+
+jobs:
+  run-snyk-monitor:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: hivemq/hivemq-checkout-composite-action@v1
+        with:
+          path: mqtt-cli
+
+      - name: Run Snyk monitor
+        uses: snyk/actions/gradle-jdk11@master
+        env:
+          SNYK_TOKEN: ${{ secrets.SNYK_TOKEN }}
+          ORG_GRADLE_PROJECT_hivemqCommonsUsername: ${{ secrets.PUBLISH_GITHUB_USERNAME }}
+          ORG_GRADLE_PROJECT_hivemqCommonsPassword: ${{ secrets.PUBLISH_GITHUB_TOKEN }}
```

Comment:
Do we really need these credentials for the mqtt-cli?

Reply:
Good catch! We don't need them.